manifests/m/Microsoft/AzureDataStudio/1.39.1/Microsoft.AzureDataStudio.locale.en-US.yaml

```diff
@@ -5,13 +5,13 @@ PackageIdentifier: Microsoft.AzureDataStudio
 PackageVersion: 1.39.1
 PackageLocale: en-US
 Publisher: Microsoft Corporation
-PublisherUrl: https://www.microsoft.com/en-us/
+PublisherUrl: https://www.microsoft.com
 PublisherSupportUrl: https://github.com/microsoft/azuredatastudio/issues
-PrivacyUrl: https://privacy.microsoft.com/en-us/privacystatement
+PrivacyUrl: https://privacy.microsoft.com

```

Comment:
This should probably stay ad a direct link to the privacy statement, rather than the top level page